src/components/ItemFilter.jsx

```diff
@@ -1,7 +1,16 @@
 import React from 'react';
 
-export const ItemFilter = ({ item }) => {
+export const ItemFilter = ({ element, selectedElement }) => {
+  const selectedStyles = selectedElement === element.en
+    ? {
+      color: 'white',
+      backgroundColor: '#11698E',
+      paddingLeft: '0.5rem',
+    }
+    : {
+      paddingLeft: '0.5rem',
+    };
```

Comment:
Since paddingLeft is in both cases, paddingLeft 0.5rem can be moved to className.

You can use the TailwindCSS utilities class for this:

![image](https://user-images.githubusercontent.com/66505715/163267504-188bf575-2c58-43e2-aa91-01c8eef09682.png)